extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigation_index.php

```diff
@@ -0,0 +1,67 @@
+<div class="wds-global-navigation">
+	<div class="wds-global-navigation__content-bar">
+		<a href="<?= Sanitizer::encodeAttribute( $model['logo']['header']['href'] ); ?>" class="wds-global-navigation__logo">
+			<?= DesignSystemHelper::getSvg(
+				$model['logo']['header']['image'],
+				'wds-global-navigation__logo-fandom'
+			) ?>
+		</a>
+		<?php
+		if ( isset( $model['fandom_overview'] ) ):
+			foreach ( $model['fandom_overview']['links'] as $link ):
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "links" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)